composer.json

```diff
@@ -9,17 +9,17 @@
         "drupal/core": "~8.6@alpha",
```

Comment:
I would set this to `~8.6` so that it is consistent with the one used for `openeuropa/drupal-core-require-dev`.